assets/app/views/directives/_status-icon.html

```diff
@@ -1,4 +1,4 @@
-<span ng-switch="status" class="hide-ng-leave status-icon">
+<span ng-switch="status | capitalize" class="hide-ng-leave status-icon">
```

Comment:
Thats fine. I think I ran into a case issue and didn't want to repeat`<status-icon status="status | capitalize"`.  I'll double check though and see if I can eliminate.